.github/workflows/test-migrations.yml

```diff
@@ -2,21 +2,46 @@ name: Test Django Migrations
 
 on:
   push:
-    branches: [ master ]
+    branches:
+      - master
   pull_request:
-    branches: [ master ]
+    branches:
+      - master
 
 jobs:
   build:
-
     runs-on: ubuntu-latest
+    env:
+      DOCKER_BUILDKIT: 1
+      COMPOSE_DOCKER_CLI_BUILD: 1
 
     steps:
+
     - uses: actions/checkout@v2
 
+    - name: Set up Docker Buildx
+      uses: docker/setup-buildx-action@v1
+
+    - name: Login to DockerHub
+      uses: docker/login-action@v1
+      with:
+        username: ${{ secrets.DOCKERHUB_USERNAME }}
+        password: ${{ secrets.DOCKERHUB_TOKEN }}
+
+    - name: Build Django image
+      uses: docker/build-push-action@v2
+      with:
+        context: .
+        file: docker/django/Dockerfile
```

Comment:
there's also the caddy Dockerfile that we must build, and maybe more in the future.

Isn't there a way to use the docker-compose setup with this approach ?

Reply:
I don't think there is an action that build the compose + push all images.
Moreover, we might not want to push all images, only this one make sense to push for now.

Reply:
ok